python/demo/main.py

```diff
@@ -14,7 +14,7 @@ def api_index() -> list[AnyComponent]:
     # language=markdown
     markdown = """\
 This site providers a demo of [FastUI](https://github.com/samuelcolvin/FastUI), the code for the demo\
```

Comment:
Maybe just removing the \ at the end here will fix it. I think with markdown one newline becomes a single whitespace you need two new lines for a br

Reply:
ye, you're right - just removing the `\` fixes it.

Do you want to update your PR, or should I fix?